### PR TITLE
added recipe for pycrs

### PR DESCRIPTION
**Pull request by snowman2**

### CODE REVIEW

**Comment by snowman2** [comment]
All tests pass and ready to merge 😀

File: recipes/pycrs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  license_file: LICENSE.txt
+  summary: 'GIS package for reading, writing, and converting between CRS formats.'
+  description: |
+    PyCRS is a pure Python GIS package for reading, writing, and converting between various common coordinate reference system (CRS) string and data source formats.
+  doc_url: https://github.com/karimbahgat/PyCRS
+  dev_url: https://github.com/karimbahgat/PyCRS
+
+extra:
+  recipe-maintainers:
+    - karimbahgat
```
Comment (marcelotrevisani) [inline recipes/pycrs/meta.yaml]: Does @karimbahgat agree to be a maintainer as well?

**Comment by snowman2** [comment]
@marcelotrevisani, thanks for the comments! The issue attached to this thread has the license mentioned as something to add in the next release. Also, @karimbahgat was planning on adding pycrs to conda-forge in the future. But, not a bad idea to wait to hear from him if he would like to be a maintainer.

**Comment by marcelotrevisani** [comment]
ping @snowman2

**Comment by snowman2** [comment]
@karimbahgat I removed you from the maintainers list due to no response. If you would like to be added, just let me know and I would be happy to add you.

**Comment by karimbahgat** [comment]
Hi. Thanks for staying on top of this! To be honest, if this has already been fixed and there’s nothing for me to do, i’d rather not be a maintainer as of right now. I dont really know the details of conda forge recipes and would have to use some time to read up on it. 
If there’s something you would need from me for the main pycrs repo please let me know, the license.txt file should be in the next release hopefully in the next week.

**Comment by djhoese** [comment]
Is there anything still holding this recipe from being merged @conda-forge/staged-recipes? Given my investment with geoxarray on the pycrs library I wouldn't mind being a maintainer of this package so @snowman2 doesn't have to be the only one.

**Comment by ocefpaf** [comment]
The best part about hanging around staged-recipes is to get to know the projects people are working on. I had no idea pycrs existed and it is such a useful package! Thanks @snowman2 for packaging it, @marcelotrevisani for the review, and @djhoese for the ping.

**Pull request merged**